docs/src/main/sphinx/connector/ignite.md

```diff
@@ -78,6 +78,18 @@ configured connector to create a catalog named `sales`.
 ```{include} non-transactional-insert.fragment
 ```
 
+### Non-transactional MERGE
```

Comment:
We should merge #24523 first and include the fragment in my opinion.

Reply:
Sure!